src/__Private/LintRun.hack

```diff
@@ -116,7 +116,7 @@ final class LintRun {
     }
 
     try {
-      $errors = await $linter->getLintErrorsAsync();
+      $errors = await $linter->getProblemsAsync();
```

Comment:
I think I missed this somewhere in the stack: this should consistently be 'Errors', not 'Problems'